src/index.css

```diff
@@ -161,6 +159,40 @@ nav a.active-link {
   display: none;
 }
 
+.terms-and-about {
+  margin-left: 16px;
+  margin-right: 16px;
```

Comment:
You have this spacing in multiple places: On mobile it's 16px left & right (sometimes only left, because it's not visible if spacing on the right is missing), on desktop it's 24px.

| Terms & About | H1 | Grid Container | Navigation |
|--------|--------|--------|--------|
| <img width="257" alt="Screenshot 2024-11-06 at 12 45 56" src="https://github.com/user-attachments/assets/9a130f4e-cb30-4e8b-bed7-8232789a1c29"> | <img width="250" alt="Screenshot 2024-11-06 at 12 46 15" src="https://github.com/user-attachments/assets/200832ac-f7cf-4c42-ad9c-d249998e6834"> | <img width="252" alt="Screenshot 2024-11-06 at 12 46 54" src="https://github.com/user-attachments/assets/d11ec5fb-4e04-4a50-ad31-36e7092615e0"> | <img width="165" alt="Screenshot 2024-11-06 at 12 47 55" src="https://github.com/user-attachments/assets/774805b3-3026-4aab-85bc-7bbbc96a9829"> <img width="166" alt="Screenshot 2024-11-06 at 12 50 50" src="https://github.com/user-attachments/assets/54ec3817-c83e-43e3-9cef-34bad67a1d2d"> |

I suggest you create a wrapper class which "centralizes" this behaviour. You can then use it in the form of a wrapper element (like the `<Container>` component in [React Bootstrap](https://react-bootstrap.netlify.app/docs/layout/grid/#container) or [MUI](https://mui.com/material-ui/react-container/) or [other component systems](https://www.chakra-ui.com/docs/components/container)) or just by adding the class to your existing elements (`className="my-class container"`, just like in the non-React [Bootstrap](https://getbootstrap.com/docs/5.0/layout/containers/) or [Tailwind CSS](https://tailwindcss.com/docs/container)).